.github/ISSUE_TEMPLATE/release-it-tests.md

```diff
@@ -0,0 +1,41 @@
+|||
+| :--                                   | :--                                                       |
+| **Main release issue**                | RELEASE_ISSUE                                             |
+| **Main release candidate issue**      | RELEASE_CANDIDATE_ISSUE_URL                               |
+| **Version**                           | WAZUH_VERSION                                             |
+| **Release candidate**                 | RC_VERSION                                                |
+| **wazuh/wazuh tag**                   | https://github.com/wazuh/wazuh/tree/vWAZUH_VERSION-TAG    |
+| **wazuh/wazuh-qa tag**                | https://github.com/wazuh/wazuh/tree/vWAZUH_VERSION-TAG    |
+| **Previous release testing issue**    | PREVIOUS_IT                                               |
+| **Build**                             | COMPLETE_THIS_FIELD :warning: :warning: :warning:         |
+
+</br>
+
+<details><summary>Parameters</summary></br>
+
+- `PKG_VERSION`: PKG_VERSION
+- `PKG_REVISION`: PKG_REVISION
+- `TARGET_REPOSITORY`: TARGET_REPOSITORY
+- `QA_REFERENCE`: QA_REFERENCE
+- `JENKINS_REFERENCE`: JENKINS_REFERENCE
+- `LINUX_OS`: LINUX_OS
```

Comment:
Just linux?

Reply:
That's the manager OS (CentOS or Ubuntu)